controllers/update_status_test.go

```diff
@@ -380,5 +380,53 @@ var _ = Describe("update_status", func() {
 				Expect(removalCount).To(BeNumerically("==", 1))
 			})
 		})
+
+		When("one instance is not reachable", func() {
+			var unreachableProcessGroup string
+
+			BeforeEach(func() {
+				unreachableProcessGroup = instances[0].GetInstanceID()
+				instances[0].Pod.Annotations[mockUnreachableAnnotation] = "banana"
```

Comment:
Shouldn't we need to clear this after test completes

Reply:
We create a complete new cluster object for each test so we don't have to remove anything.